src/App.tsx

```diff
@@ -15,48 +15,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabeticaly = 'Alphabeticaly',
+  Length = 'Length',
+}
+
+interface SortParametrs {
+  typeOfSort: string;
```

Comment:
You alraedy have types for sort order
```suggestion
  typeOfSort: SortType
```